b2sdk/b2http.py

```diff
@@ -120,6 +120,9 @@ def _translate_and_retry(fcn, try_count, post_params=None):
         except B2Error as e:
             if not e.should_retry_http():
                 raise
+            if hasattr(e, 'retry_after_seconds') and e.retry_after_seconds:
```

Comment:
```suggestion
            if hasattr(e, 'retry_after_seconds') and e.retry_after_seconds is not None:
```
because the server could for some reason tell us to retry immediately